src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -173,6 +174,8 @@ private static Task StartProcessWithKeepAlive(ProcessMetadata metadata)
             return Task.Run(
                 () =>
                 {
+                    Thread.Sleep(10_000);
```

Comment:
First I've seen the underscore used as a separator. Good C#7 feature!

Reply:
Makes it much more readable. I saw someone else do it once and loved it.